src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+      <!-- Temporarily disable public API analyzers until API has been stabilized -->
+    <NoWarn>RS0016,RS0037,$(NoWarn)</NoWarn>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
+  </PropertyGroup>
+
+    <ItemGroup>
```

Comment:
[Nit] I would reformat the file; the `ItemGroup` node's indent is off by one tab currently.

Reply:
Oh good eye